app/tasks/maintenance_tasks/task.rb

```diff
@@ -103,13 +123,35 @@ def load_constants
       end
     end
 
+    # The contents of a CSV file to be processed by a Task.
+    #
+    # @return [String] the content of the CSV file to process.
+    def csv_content
+      raise NoMethodError unless has_csv_content?
+
+      @csv_content
+    end
+
+    # Set the contents of a CSV file to be processed by a Task.
+    #
+    # @param csv_content [String] the content of the CSV file to process.
+    def csv_content=(csv_content)
```

Comment:
This can be made generic using method missing to allow the strategy to define which attributes can be set. Of course that would be meta programming heavy, so harder to understand since we need to set instance variables that don't exist using `instance_variable_set`.